src/components/Rankings.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { getAllFieldCoveredCounts } from '../lib/coverageStats/statsUtils';
+
+const SectionColors = {
+  Patient: 'fill-[#D24200]',
+  Outcome: 'fill-[#8A45D9]',
+  Disease: 'fill-[#F2B84B]',
+  Treatment: 'fill-[#04B2D9]',
+  Assessment: 'fill-[#F2913D]',
+  Genomics: 'fill-[#26C485]',
+};
```

Comment:
Let's also make this table an import from a lib file, since we will definitely be referencing it across multiple files.